packages/sanity/src/core/form/studio/inputs/globalDocumentReference/datastores/getReferenceInfo.ts

```diff
@@ -78,12 +73,25 @@ export function createGetReferenceInfo(context: {client: ReferenceClient}) {
         }
 
         const previewPaths = getPreviewPaths(refSchemaType?.preview) || []
-        const listener = client.listen('*', {}, {includeResult: true}).pipe(share())
+        const listener = client.observable.listen(
+          '*',
+          {},
+          {
+            includeResult: true,
+            tag: `${REQUEST_TAG_BASE}.preview`,
+            events: ['mutation', 'welcome'],
```

Comment:
non-obvious to me why we don't care about other events. add a code comment explaning why?

Reply:
yesss 👍